catchup/catchpointService.go

```diff
@@ -803,3 +812,26 @@ func (cs *CatchpointCatchupService) initDownloadPeerSelector() {
 			})
 	}
 }
+
+// checkLedgerDownload sends a HEAD request to the ledger endpoint of peers to validate the catchpoint's availability
+// before actually starting the catchup process.
+// The error returned is either from an unsuccessful request or a successful request that did not return a 200.
+func (cs *CatchpointCatchupService) checkLedgerDownload() error {
+	round, _, err := ledgercore.ParseCatchpointLabel(cs.stats.CatchpointLabel)
+	if err != nil {
+		return fmt.Errorf("failed to parse catchpoint label : %v", err)
+	}
+	peerSelector := makePeerSelector(cs.net, []peerClass{{initialRank: peerRankInitialFirstPriority, peerClass: network.PeersPhonebookRelays}})
```

Comment:
why there is a second peer selector created with a different peerClass in addition to one in make service?